### PR TITLE
Replace node with blog in URL

### PR DESCRIPTION
**Pull request by Vandit1604**
Replaced occurrences of `node` with `blog` in `content/` dir of blog pages.

### CODE REVIEW

**Comment by krisstern** [comment]
Hi @Vandit1604, thanks for this PR. But would it be possible to add redirection from `node` to `blog` just in case?

**Comment by Vandit1604** [comment]
I can't remember if there's no URL with `node`. Perhaps I missed something. Could you please point to the URL that contains the word `node`?

**Comment by krisstern** [comment]
There is no longer any reference of `node` in your implementation. But there could still be someone with the old URL, so we will need to be careful and redirect anyways while migrating from the old site.

**Comment by Vandit1604** [comment]
Checked Gatsby's documentation to see if they support redirections. I found this: https://www.gatsbyjs.com/docs/how-to/cloud/working-with-redirects-and-rewrites/#splat-redirects.

I implemented this in https://github.com/Vandit1604/jenkins-docs/pull/112/commits/17096b691c263d9e7cbcf34e5cef757cc5640110 but did not work. Furthermore, I believe it only works on Gatsby Cloud or Netlify.
I changed my approach. I'm looking into https://www.gatsbyjs.com/plugins/gatsby-plugin-htaccess/.

**Comment by krisstern** [comment]
> Checked Gatsby's documentation to see if they support redirections. I found this: https://www.gatsbyjs.com/docs/how-to/cloud/working-with-redirects-and-rewrites/#splat-redirects.
> 
> I implemented this in [17096b6](https://github.com/Vandit1604/jenkins-docs/commit/17096b691c263d9e7cbcf34e5cef757cc5640110) but did not work. Furthermore, I believe it only works on Gatsby Cloud or Netlify. I changed my approach. I'm looking into https://www.gatsbyjs.com/plugins/gatsby-plugin-htaccess/.

@Vandit1604 Did you tried adding `force: true` to `createRedirect`?

**Comment by Vandit1604** [comment]
I'll try it

**Comment by Vandit1604** [comment]
It didn't work. Do you want me to commit the changes for you to take a look?

**Comment by krisstern** [comment]
> It didn't work. Do you want me to commit the changes for you to take a look?

Sure

**Comment by Vandit1604** [comment]
```js
  createRedirect({
    fromPath: '/node/',
    toPath: '/blog/',
    exactPath: true,
    isPermanent: false,
    redirectInBrowser: true,
    force: true,
  });
  ```
  
  With this, Now the URL changes from `node` to `blog` but the page doesn't actually load or reload?
  I'll see if i can fix that and then commit my changes.

**Comment by krisstern** [comment]
> ```js
> isPermanent: false,
> ```

How about changing `isPermanent: false,` to `isPermanent: true,` also?

**Comment by Vandit1604** [comment]
Nothing changed. The URL is changing to `/blog` but the page isn't reloading or loading.

I also tried https://github.com/gatsbyjs/gatsby/blob/master/examples/using-redirects/ but on redirects the page is not loading in that example too.

**Comment by Vandit1604** [comment]
I think this is the best we can do. if this doesn't work when we host the blog site we can use the .htacess file to define the redirect.

**Review by krisstern — APPROVED** [review]
LGTM

File: site/gatsby-node.js [hunk 1]
```diff
@@ -8,7 +8,25 @@ const { createFilePath } = require(`gatsby-source-filesystem`);
...
+    force: true,
+  });
+
+  createRedirect({
+    fromPath: `/node`,
+    toPath: `/node/`,
+    isPermanent: true,
+    redirectInBrowser: true,
+    force: true,
+  });
```
[the remark this answers is not in the document]
Comment (Vandit1604) [reply site/gatsby-node.js:28]: Here, We can redirect `/node` to `/blog/` here.

[the remark this answers is not in the document]
Comment (krisstern) [reply site/gatsby-node.js:28]: But does it take care of both `/node` and `/node/` redirecting to `/blog/`?

**Pull request merged**